linkerd/app/src/env.rs

```diff
@@ -1017,6 +1020,24 @@ mod tests {
         assert_eq!(parse_duration("1"), Err(ParseError::NotADuration));
     }
 
+    #[test]
+    fn convert_labels_to_map_different_values() {
```

Comment:
It seems like it might also be good to test some malformed/invalid labels, and ensure that they are ignored rather than panicking?